src/main/java/jenkins/plugins/rpmsign/GpgKey.java

```diff
@@ -35,8 +35,8 @@ public String getId() {
 
   public int getUniqueId() {
     int result = name != null ? name.hashCode() : 0;
-    result = 31 * result + (privateKey.getPlainText() != null ? privateKey.getPlainText().hashCode() : 0);
-    result = 31 * result + (passphrase.getPlainText() != null ? passphrase.getPlainText().hashCode() : 0);
+    result = 31 * result + privateKey.getPlainText().hashCode();
+    result = 31 * result + passphrase.getPlainText().hashCode();
```

Comment:
Why remove this null check?

Reply:
It can not be null. 
getPlainText() method of Secret class uses value field and it is annotated as @Nonnull.